pages/common/go-build.md

```diff
@@ -3,7 +3,7 @@
 > Compile Go sources.
 > More information: <https://golang.org/cmd/go/#hdr-Compile_packages_and_dependencies>.
 
-- Compile a file:
+- Compile a file (output will be filename without extension):
```

Comment:
I think this is not always the case. It only will produce a binary if we are compiling the `main` package.
```suggestion
- Compile a 'package main' file (output will be filename without extension):
```